instrumentation/database/sql/splunksql/test/suite_test.go

```diff
@@ -307,6 +308,7 @@ func (s *SplunkSQLSuite) assertSpans(name moniker.Span, count int, c traceapi.Sp
 			}
 		}
 		s.Equalf(traceapi.SpanKindClient, roSpan.SpanKind(), "span %q is not a client span", name)
+		s.Equal(splunkotel.Version(), roSpan.InstrumentationLibrary().Version, "version should match")
```

Comment:
Shouldn't this test against the version from `instrumentation/database/sql/splunksql`?

And similar for the other `test` packages?

Reply:
I think we can just export `Version`. That is what we do upstream when we need to reference the version like this.

I would refrain from adding a version to the test packages as they are only intended to test their respective instrumentation.

Reply:
How about addressing it in a separate PR?  I can create an issue for tracking. I plan to solve it on Tuesday.

Reply:
The good thing about doing it in a separate PR is that this PR will be a fix and the next one would expand the API.

Reply:
https://github.com/signalfx/splunk-otel-go/issues/1992